skl2onnx/operator_converters/support_vector_machines.py

```diff
@@ -140,6 +148,127 @@ def convert_sklearn_svm(scope, operator, container):
         raise ValueError("Unknown support vector machine model type found "
                          "'{0}'.".format(operator.type))
 
+    if (hasattr(op, 'decision_function_shape') and
+            op.decision_function_shape == 'ovr' and handles_ovr):
+        # Applies _ovr_decision_function.
+        # See https://github.com/scikit-learn/scikit-learn/blob/
+        # master/sklearn/utils/multiclass.py#L407:
+        # ::
+        #     _ovr_decision_function(dec < 0, -dec, len(self.classes_))
+        #
+        #     ...
+        #     def _ovr_decision_function(predictions, confidences, n_classes):
+        #
+        #     n_samples = predictions.shape[0]
+        #     votes = np.zeros((n_samples, n_classes))
+        #     sum_of_confidences = np.zeros((n_samples, n_classes))
+        #     k = 0
+        #     for i in range(n_classes):
+        #         for j in range(i + 1, n_classes):
+        #             sum_of_confidences[:, i] -= confidences[:, k]
+        #             sum_of_confidences[:, j] += confidences[:, k]
+        #             votes[predictions[:, k] == 0, i] += 1
+        #             votes[predictions[:, k] == 1, j] += 1
+        #             k += 1
+        #     transformed_confidences = (
+        #         sum_of_confidences / (3 * (np.abs(sum_of_confidences) + 1)))
+        #     return votes + transformed_confidences
+        if list(op.classes_) != list(range(len(op.classes_))):
+            raise RuntimeError(
+                "Classes different from first n integers are not supported "
+                "in SVC converter.")
+
+        cst3 = scope.get_unique_variable_name('cst3')
```

Comment:
It's difficult to understand what cst0, cst1 and cst3 mean. It would be better to have clearer node names.

Reply:
cst0 = constant equal to 0, cst1 = constant equal to 1, cst3 = constant equal to 3

Reply:
Variable name is fine, it would be better to have the onnx node name as `const0, etc.` Anyways, this is a minor issue.